### DISCUSSION TITLE
ETA for the first previews of v3

### DISCUSSION THREAD

**Discussion by umbarov** [xunit/xunit#2196] [Question] [answered]
Hi,
Is there any release date for the v3 previews?

**Comment by bradwilson** [comment, marked as answer]
Not at this time. I'm still working through the [roadmap](https://github.com/xunit/xunit/issues/2133), and in particular I need to finish the item under Runner Library titled "Support for new v3 cross-process RPC" so that external runners (like support for VSTest, meaning `dotnet test` and Visual Studio Test Explorer) can run v3 tests.
